### PR TITLE
[android] Fixed disabling of locals ads on the start if subscription …

### PR DESCRIPTION
**Pull request by alexzatsepin**
…is active
https://jira.mail.ru/browse/MAPSME-8642

### CODE REVIEW

**Comment by alexzatsepin** [comment]
@darina PTAL

File: map/framework.cpp [hunk 1]
```diff
@@ -481,8 +481,8 @@ Framework::Framework(FrameworkParams const & params)
   // Local ads manager should be initialized after storage initialization.
   if (params.m_enableLocalAds)
   {
-    m_localAdsManager.Startup(m_bmManager.get(),
-      m_purchase->IsSubscriptionActive(SubscriptionType::RemoveAds));
+    bool isActive = m_purchase->IsSubscriptionActive(SubscriptionType::RemoveAds);
```
Comment (darina) [inline map/framework.cpp]: auto const isActive

Comment (alexzatsepin) [reply map/framework.cpp]: fixed

**Comment by alexzatsepin** [comment]
@darina PTAL

**Pull request merged**